metadata-ingestion/docs/sources/azure-data-factory/azure-data-factory_pre.md

```diff
@@ -0,0 +1,209 @@
+## Overview
+
+This connector extracts metadata from Azure Data Factory (ADF), including:
+
+- **Data Factories** as Containers
```

Comment:
Not sure why, but this is rendering as a code block on the docs site. Try indenting the bullet points and/or using `*` instead of `-` for aech bullet